db/models.py

```diff
@@ -50,3 +52,69 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        to="db.User", on_delete=models.CASCADE, related_name="orders"
+    )
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+    class Meta:
+        ordering = ["-created_at"]
+
+
+class Ticket(models.Model):
+    row = models.IntegerField()
+    seat = models.IntegerField()
+    movie_session = models.ForeignKey(
+        to="db.MovieSession", on_delete=models.CASCADE, related_name="tickets"
```

Comment:
Ensure that the reference to 'db.MovieSession' matches the actual app label and model name in your Django project. If the app label is different, update it accordingly.